### DISCUSSION TITLE
[Bug] Controller not detected

### DISCUSSION THREAD

**Discussion by Sharkblack3D** [redphx/better-xcloud#612] [Questions]
### Checklist

- [X] I will only use English in my report.
- [X] I have used the search function for [**open and closed issues**](https://github.com/redphx/better-xcloud/issues?q=is%3Aissue) to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### xCloud officially supports your country/region

Yes

### The bug doesn't happen when you disable Better xCloud script

No

### Previous Better xCloud versions didn't have this bug (name which one)

Yes

### Device

Other

### Device

JMGO N1S Pro Projecteur Video

### Operating System

Android TV 11

### Android app/Browser Version

Android TV 11

### Better xCloud Version

0.22.2

### Reproduction Steps

hello,
I have try to first time your app it looks really complete only problem my official xbox controller is not detected by bluetooth or even by usb, can you help me

**Comment by Sharkblack3D** [comment]
I forgot to specify the controller works and is detected by my rear projector as an XBOX controller but only the bettercloud application which cannot detect it

**Reply by Sharkblack3D** [reply]
Hello my problem is not résolve

**Comment by Sharkblack3D** [comment]
> ### Checklist
> 
> - [X] I will only use English in my report.
> - [X] I have used the search function for [**open and closed issues**](https://github.com/redphx/better-xcloud/issues?q=is%3Aissue) to see if someone else has already submitted the same bug report.
> - [X] I will describe the problem with as much detail as possible.
> 
> ### xCloud officially supports your country/region
> 
> Yes
> 
> ### The bug doesn't happen when you disable Better xCloud script
> 
> No
> 
> ### Previous Better xCloud versions didn't have this bug (name which one)
> 
> Yes
> 
> ### Device
> 
> Other
> 
> ### Device
> 
> JMGO N1S Pro Projecteur Video
> 
> ### Operating System
> 
> Android TV 11
> 
> ### Android app/Browser Version
> 
> Android TV 11
> 
> ### Better xCloud Version
> 
> 0.22.2
> 
> ### Game list
> 
> _No response_
> 
> ### Reproduction Steps
> 
> hello,
> 
> I have try to first time your app it looks really complete only problem my official xbox controller is not detected by bluetooth or even by usb, can you help me
> 
> ### Screenshot/video
> 
> _No response_

Hi I up my discussion

**Comment by lexiismadd** [comment]
Did you figure it out? I'm having the same issue
